Testing/functional/e2e/cypress/fixtures/ImmunizationService/immunization.json

```diff
@@ -1,499 +1,504 @@
+            "lotNumber": "LOT A",
+            "productName": "Product A"
+          }
+        ]
+      },
+      {
+        "id": "3208887_54397127",
+        "name": "Measles, Mumps, Rubella",
+        "dateOfImmunization": "2005-10-05T00:00:00",
+        "providerOrClinic": "Provider A",
+        "immunizationAgents": [
+          {
+            "code": "61153008",
+            "name": "MMR",
+            "lotNumber": "LOT A",
+            "productName": "Product A"
+          }
+        ]
+      },
+      {
+        "id": "3208887_54397128",
+        "name": "Hepatitis A, Hepatitis B",
+        "dateOfImmunization": "2020-03-02T00:00:00",
+        "providerOrClinic": "Provider A",
+        "immunizationAgents": [
+          {
+            "code": "333702001",
+            "name": "HAHB",
+            "lotNumber": "LOT A",
+            "productName": "Product A"
+          }
+        ]
+      },
+      {
+        "id": "3208887_54397129",
+        "name": "Cholera",
+        "dateOfImmunization": "2020-03-02T00:00:00",
+        "providerOrClinic": "Provider A",
+        "immunizationAgents": [
+          {
+            "code": "413830002",
+            "name": "Cholera-Ecol-O",
+            "lotNumber": "LOT A",
+            "productName": "Product A"
+          }
+        ]
+      },
+      {
+        "id": "3208887_54397130",
+        "name": "Pneumococcal Conjugate - PCV13",
+        "dateOfImmunization": "2020-03-02T00:00:00",
+        "providerOrClinic": "Provider A",
+        "immunizationAgents": [
+          {
+            "code": "BCYSCT_AG047",
+            "name": "Pneumo-C-13",
+            "lotNumber": "LOT A",
+            "productName": "Product A"
+          }
+        ]
+      },
+      {
+        "id": "3208884_54396880",
+        "name": "Influenza-Inactivated",
+        "dateOfImmunization": "2019-11-20T00:00:00",
+        "providerOrClinic": "Gateway",
+        "immunizationAgents": [
+          {
+            "code": "BCYSCT_AG077",
+            "name": "Influenza-Inactivated",
+            "lotNumber": "299837",
+            "productName": "Pfizer"
+          }
+        ]
+      },
+      {
+        "id": "3208890_54396881",
+        "name": "Tetanus, Diphtheria",
+        "dateOfImmunization": "2019-11-20T00:00:00",
+        "providerOrClinic": "Provider A",
+        "immunizationAgents": [
+          {
+            "code": "59999009",
+            "name": "Td",
+            "lotNumber": "LOT A",
+            "productName": "Product A"
+          }
+        ]
+      },
+      {
+        "id": "890700884_54396874",
+        "name": "COVID-19",
+        "dateOfImmunization": "2020-01-22T00:00:00",
+        "providerOrClinic": "Provider A",
+        "immunizationAgents": [
+          {
+            "code": "407737004",
+            "name": "Zoster",
+            "lotNumber": "LOT A",
+            "productName": "Product A"
+          }
+        ]
+      }
+    ]
+  },
   "totalResultCount": 35,
```

Comment:
This is the wrong data.. the current data has 34 records.

Reply:
fixed :-)